data/projects/implement_metrics_for_coala.md

```diff
@@ -1,34 +1,46 @@
-There is more to software quality than just passing builds -
-coala should support generating metrics for your code.
-
-A wide variety of design principles influence software quality, and some of
-them can be measured objectively by so-called
-[software metrics](https://en.wikipedia.org/wiki/Software_metric). coala
-offers a good base for the implementation or wrapping of algorithms that
-calculate these metrics from a piece of code.
-
-The goal of this project proposal should be to implement handling and
-display of software metrics in coala. To prove the concept, this project
-has to include multiple bears that calculate metrics for different
-programming languages.
+There is more to software quality than just passing builds. A wide variety
+of design principles influence software quality, and some of them can be
+measured objectively by so-called
+[software metrics](https://en.wikipedia.org/wiki/Software_metric).
+Software metrics are important because they produce quantifiable measurements
+at different stages of software development which have numerous valuable
+applications in schedule and budget planning, cost estimation, quality
+assurance testing, software debugging, software performance optimization, and
+optimal personnel task assignments. They are crucial when it comes to quality
+assurance because source code metrics for quality assurance aims at writing
+maintainable code with less bugs. coala offers a good base for the
+implementation or wrapping of algorithms that calculate these metrics from a
+piece of code. The goal of this project is to implement a module in coala that
+can produce software metrics.
+
+Picking the right metrics is important and this project aims at implementing
+an API along with a set of Bears for handling and display of software metrics
+in coala.
 
 #### Milestones
 
-##### GSOC 2017 COMMUNITY BONDING
+##### PREPARATION/BONDING
 
+* Tools for calculating software metrics should be researched.
```

Comment:
Again >.>

This is not a milestone that can be checked. The milestone should describe the desired state at the end of the period. 
And it has to be testable. Testing if something is researched is not really easy. You can have the student to propose a list that includes reasoning. But than the milestone is the existence of said list.